docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.2"
+services:
+  api:
+    build: .
+    ports:
+      - "$API_HOST_PORT:8081"
+    environment:
```

Comment:
Are we switching to a .env file instead?
If so, we should update that and include an example file.
Additional environment variables are needed than these three, take a look at the startup script used by CircleCI [start-server.sh](https://github.com/overture-stack/ego/blob/develop/src/main/resources/scripts/start-server.sh)

Reply:
updated with the other variables and matched the names in the start script.
Also added a .env.schema file.